app/qml/components/MMCodeScanner.qml

```diff
@@ -13,18 +13,23 @@ import QtMultimedia
 
 import mm 1.0 as MM
 
-Drawer { // We can keep this one as Drawer - it could actually be Popup instead
+Popup {
   id: root
 
+  parent: Overlay.overlay
+  visible: true
+  height: ApplicationWindow.window?.height ?? 0
+  width: ApplicationWindow.window?.width ?? 0
+  closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutside
+
   signal scanFinished( var data )
 
-  width: ApplicationWindow.window.width
-  height: ApplicationWindow.window.height + 40 * __dp
-  edge: Qt.BottomEdge
-  dim: true
-  interactive: false
-  dragMargin: 0
-  closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutside
+  function unload() {
+    qrcodeScanner.videoSink = null
+    camera.active = false
+    captureSession.videoOutput = null
+    captureSession.camera = null
+  }
```

Comment:
Functions should be declared at the bottom of the file according to the code standard (https://github.com/MerginMaps/mobile/blob/master/docs/code_convention.md)